README.md

```diff
@@ -1 +1,61 @@
-# microverse-capstone-3
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+![Pokémon Logo](Pokémon_logo.png)
+
+# Pokémon App
+
+> This is a web application to retrieve metrics from an API using React and Redux.
+> The requirements include selecting an API that provides numeric data about a topic and then building the web app around the topic. The web app should have:
+> - a home page with a list of items
+> - another page for the item details
+
+## Learning Objectives
+- Use React documentation
+- Use React components
+- Use React props
+- Use React Router
+- Connect React and Redux
+- Handle events in a React app
+- Write integration tests with a React testing library
+- Use styles in a React app
+- Use React life cycle methods
+- Apply React best practices and language style guides in code
+- Use store, actions and reducers in React
+
+## Project Presentation
+- [Video link](https://www.loom.com/share/de9c94bfa0bc4c30a837434505cecb23)
+
+## Built With
+- Languages: HTML5, CSS3, JavaScript
+- Front-end library: React & Redux
+- API
+
+## Live Demo
+Live demo on [netlify]()
+
+## Getting Started
+To get a local copy up and running follow these steps
+- Clone the repository using the GitHub link provided below.
+- In your terminal, run: <code>git clone git@github.com:DJ-MrJay/microverse-capstone-3.git</code>
+- Run tests. Follow these [instructions](https://github.com/microverseinc/linters-config). A GitHub action is also set to run during pull request
```

Comment:
- [ ] I think the README file is updated so kindly update it to the provided template from microverse you can check this link and please update all the sections to make it descriptive and professional 🚀✨💯🔥 [Readme](https://github.com/microverseinc/readme-template)